extension/dynamicconfig/service/localconfig.go

```diff
@@ -0,0 +1,115 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"bytes"
+	"fmt"
+	"sync"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/spf13/viper"
+	"go.opentelemetry.io/collector/config"
```

Comment:
Another nit: Would it be neater to sort imports alphabetically or something? Like group github imports together, etc.

Reply:
good point, I've tried to keep them alphabetical within groups. I used a "python style" (PEP 8) import grouping -- standard library, third-party library, then local -- though I'm not sure what the convention might be in go?